### DISCUSSION TITLE
remoteEntry PoC - what is the minimal start test to see if the federation plugin is working?

### DISCUSSION THREAD

**Discussion by xparagon** [originjs/vite-plugin-federation#182] [Q&A]
> I found this to be asked and answered (kinda) in https://github.com/originjs/vite-plugin-federation/discussions/172

If I make a minimal config, e.g. like this:

```
export default defineConfig({
  server: {
    port: 3030
  },
  plugins: [react(),
    federation({
      name: 'test-app',
      filename: 'remoteEntry.js',
      exposes: {
        test: './src/App.tsx',
        // './test': './src/App.tsx',
      },
    })
  ]
})
```

then should I not get a result from 
http://localhost:3030/remoteEntry.js

This feels like a good first step - but I can not get it working

**npm list**
├── @originjs/vite-plugin-federation@1.1.5
├── @types/react-dom@18.0.3
├── @types/react@18.0.8
├── @vitejs/plugin-react@1.3.2
├── react-dom@18.1.0
├── react@18.1.0
├── typescript@4.6.4
└── vite@2.9.7

**Comment by Jiannan-dev** [comment]
1. The first thing I want to check is whether the remote side is using vite's dev mode, which is not supported on the remote side and can only be used in build mode.
2. Another is that the url (http://localhost:3030/remoteEntry.js) is accurate, if you do not use nginx forwarding and other features url will generally contain a path such as `dist/assets`

**Comment by Jiannan-dev** [comment]
Here's an example that might help you 
https://github.com/originjs/vite-plugin-federation/tree/main/packages/examples

**Reply by siva-kannan3** [reply]
@xparagon here is a basic federated application with react & vite I have made. 
https://github.com/siva-kannan3/module-federation-vite-example
